cmd/system.go

```diff
@@ -15,12 +15,20 @@ var (
 	deployConfig    cfg.DeployConfig
 )
 
+const (
```

Comment:
👍 
Do we have these env vars documented somewhere?
Also, you said we have an order-of-precedence of _env vars > CLI flags > yaml config_, might be worth documenting too